src/dune_pkg/opamUrl0.ml

```diff
@@ -18,3 +18,5 @@ let is_version_control t =
   | #version_control -> true
   | _ -> false
 ;;
+
+let is_file t = String.equal t.transport "file"
```

Comment:
Will this work for `git+file` as well?

Reply:
Maybe eventually. Do we currently have any support for using a git repo as a package source (other than treating it as a directory)? Do we plan to support that as part of the pkg mvp?

Reply:
I suppose it's needed to pin packages to local git repos. Sounds pretty useful, but I wouldn't be too worried if it wasn't available in the MVP.